src/initializeInpageProvider.ts

```diff
@@ -12,6 +13,11 @@ type InitializeProviderOptions = {
    */
   connectionStream: Duplex;
 
+  /**
+   * The EIP-6963 provider info that should be announced if set.
+   */
+  shouldAnnounceProviderInfo?: EIP6963ProviderInfo;
```

Comment:
IIRC this is contrary to our naming conventions since it's not a boolean, and we ought to change it to:

```suggestion
  /**
   * The EIP-6963 provider info that should be announced if set.
   */
  providerInfo?: EIP6963ProviderInfo;
```

Reply:
done: https://github.com/MetaMask/providers/pull/263/commits/c1d5dc6af5db4f8d1c8c2970815f5e7e339ead08